src/addons/Confirm/Confirm.js

```diff
@@ -14,46 +14,29 @@ export default class Confirm extends Component {
     className: PropTypes.string,
     confirmLabel: PropTypes.string,
     header: PropTypes.string,
-    ref: PropTypes.string,
   }
 
   static defaultProps = {
     abortLabel: 'Cancel',
     confirmLabel: 'Yes',
-    ref: 'modal',
```

Comment:
There is no reason to set an internal access only `ref` name via props.  Anyway, it was not used.